pom.xml

```diff
@@ -69,6 +69,7 @@
         <module>services</module>
         <module>ui</module>
         <module>ear</module>
+        <module>tools/graph-tool</module>
```

Comment:
Is it included in standard build on purpose or we use it in a profile to build it only on a request from developer to debug it?